Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -40,13 +40,25 @@ namespace OpenInfraPlatform {
 	namespace Core {
 		namespace IfcGeometryConverter {
 			class ItemData;
-
+			
+			/*! \brief Converter functionality for \c IfcSurfaces's subtypes.
+            		*
+          		* This class includes converter functions for
+            		* \c IfcBoundedSurface and its subtypes,
+            		* \c IfcElementarySurface and its subtypes and
+           	 	* \c IfcSweptSurface.	

```

Comment:
I think there are also IfcFace converters. And I also added some from the representationConverter (see #147 ).